app/src/sharedTest/java/org/oppia/app/activity/HtmlParserTestActivty.kt

```diff
@@ -50,8 +52,8 @@ class HtmlParserTestActivityTest {
 
     val textView = activityTestRule.activity.findViewById(R.id.test_html_content_text_view) as TextView
     val htmlResult: Spannable = HtmlParser(
-      ApplicationProvider.getApplicationContext(), "", ""
-    ).parseOppiaHtml(
+      ApplicationProvider.getApplicationContext(), /* entityType= */ "",  /* entityId= */"",
```

Comment:
Nit: space between ``*/`` and ``""``. Ditto elsewhere.

Reply:
corrected.